mozilla/blurts-server
# MNTOR-3272: API to call FxA apply coupon

# References:

Jira: MNTOR-3272



# Description
- service-side function to apply coupon code for the cancellation flow
- a new ENV VAR to set the coupon code ID

# How to test
In Heroku, subscribe to monthly, make sure the user is subscribed.
Go to settings page and hit cancel

# Checklist (Definition of Done)
